specifyweb/frontend/js_src/lib/components/SearchDialog/index.tsx

```diff
@@ -60,14 +60,15 @@ export function SearchDialog<SCHEMA extends AnySchema>(props: {
   readonly onClose: () => void;
   readonly searchView?: string;
```

Comment:
(optional) instead of adding new prop, we could allow passing `false` for `searchView` to explicitly opt out of search view being shown